style.css

```diff
@@ -1,7 +1,37 @@
+
 .header{
 	font-size: 36px;
 	letter-spacing: 1.2px;
 	font-family: helvetica;
 	color:#536240;
 	font-weight: 700;
+}
+.info{
+	position: relative;
+	display: flex;
+	left: 800px;
+	bottom:90px;
+	flex-direction: row;
+
+}
+.vl{
+	border-left:1px solid black;
```

Comment:
Instead of adding a class, try to use the border property, for example any element has a border-right property, add border-right: 1px solid black, and you will have the same output :) Also try to avoid using string colors, so instead of black try to use the hexa value, for black it will be #000